QR-dtg.py

```diff
@@ -1,13 +1,14 @@
-import base64, os, platform, re, requests, time
+import base64, os, platform, re, time, sys
+from requests import get
 from bs4 import BeautifulSoup
 from colorama import Fore, init
 from PIL import Image
 from selenium import webdriver
 from warnings import filterwarnings
+from discord_webhook import DiscordEmbed, DiscordWebhook
```

Comment:
A good discord webhook module for sending with webhook. Also allow use of file, embed etc...